stories/topics/tech-note-gcp-extensionnodes.adoc

```diff
@@ -0,0 +1,9 @@
+[id="tech-note-gcp-extensionnodes"]
+
+= Extension Nodes
+
+{AAPonGCP} can be extended by adding an extension nodes after you have purchased and launched an extension from public or private offer.
+
+*Main Deployment Name* is a required field when you deploy an extension node from marketplace. 
```

Comment:
Perhaps say where it's a required field? Maybe a cross reference to the relevant section?